src/shared/components/case-viewer/services/case.resolver.ts

```diff
@@ -94,8 +110,17 @@ export class CaseResolver implements Resolve<CaseView> {
   private checkAuthorizationError(error: any) {
     // TODO Should be logged to remote logging infrastructure
     console.error(error);
+    if (error.status === 400 && this.requestOriginatedFrom === CaseResolver.REQUEST_ORIGINATED_FROM) {
+      this.router. navigate([CaseResolver.NO_RESULTS_PAGE_URL], {
```

Comment:
@johnbenjamin-hmcts - there's an extra space between `.` and `navigate`.

Reply:
Thank you for spotting this @Daniel-Lam, I have now fixed this.